Dockerfile

```diff
@@ -2,6 +2,24 @@ FROM starefossen/github-pages:112
 
 ENV VERSIONS="v1.4 v1.5 v1.6 v1.7 v1.8 v1.9 v1.10 v1.11 v1.12"
```

Comment:
I'd put every docs/branch related ENV-vars after the installation of NGINX; we could then even decide to create a base-image containing Jekyll and NGINX